Formula/kawa.rb

```diff
@@ -1,32 +1,26 @@
 class Kawa < Formula
   desc "Programming language for Java (implementation of Scheme)"
   homepage "https://www.gnu.org/software/kawa/"
-  url "https://ftpmirror.gnu.org/kawa/kawa-2.2.tar.gz"
-  mirror "https://ftp.gnu.org/gnu/kawa/kawa-2.2.tar.gz"
-  sha256 "c3e2cb5ae772e7441ac31484083bcee651de941bbfed5dbe4874964839b9ba32"
-
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "6835c16df5b606317d824e24fa77a6ebbcba2d84bf65f3e4c7651e78d19b265d" => :sierra
-    sha256 "4597b613cb09de8c983842bb9a430165c3f573e1721759ee6522acd3b62fe957" => :el_capitan
-    sha256 "e90e2696e371914074e8a23bf787f5f490cbdce83ec87d8ca00034c9dd541386" => :yosemite
-  end
+  url "https://ftpmirror.gnu.org/kawa/kawa-2.2.zip"
+  mirror "https://ftp.gnu.org/gnu/kawa/kawa-2.2.zip"
+  sha256 "eb2b8301786d5e983769c2e29ba7caf69afcb14977eac16d544471b9a3c79f79"
 
   depends_on :java
```

Comment:
Yeah, I see from the release notes that since Kawa 2.0, it targets Java 7.

From the release notes:

> The default is now Java 7, rather than Java 6. This means the checked-in source code is pre-processed for Java 7, and future binary releases will require Java 7.